modules/hcp-ibmz-adding-credentials-registry.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-adding-credentials-registry_{context}"]
+= Adding Credentials and Registry Certificate Authority to the Management Cluster
+
+To pull the mirror registry images from the Management Cluster , you must first add Credentials and Registry Certificate Authority of mirror registry to the Management cluster. Follow the procedure below:
+
+. Create a `ConfigMap` with the certificate of the mirror registry by running the following command:
+
++
+[source,terminal]
+----
+$ oc apply -f registry-config.yaml
+----
++
+.Example registry-config.yaml file
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: registry-config
+  namespace: openshift-config
+data:
+  <mirror_registry>: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+----
+. Patch the cluster-wide object `image.config.openshift.io` to include the following entries:
+
++
+[source,yaml]
+----
+spec:
+  additionalTrustedCA:
+    - name: registry-config
+----
+. Update the Management Cluster pull secret to add the credentials of the mirror registry.
+..  Fetch the exisiting pull secret from the cluster in a JSON format using the following command:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config -o json | jq -r '.data.".dockerconfigjson"' | base64 -d > authfile
+----
+.. Edit the fetched secret JSON file to include a section with the credentials of the Registry Certificate Authority: 
++
+[source,terminal]
```

Comment:
In the following section, if the user needs to replace any variables, add callouts to explain what they should replace.

Reply:
Checked with the team, its a secret pull and we do not want to expose it, at the same time, user doesnt hv to enter these values manually, so removing the <>, to indicate that there is no user action required here.

Reply:
Done